.github/workflows/preview.yml

```diff
@@ -0,0 +1,68 @@
+name: Publiceer PR preview voor Digi Handboek
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
```

Comment:
zie hier de closed clausule bij, maar in alle job steps `if: github.event.action != 'closed'`, is dit dan nodig?

Reply:
update: ik zie bij de laatste actie geen if clausule. wordt het daar opgeruimd?